core/src/test/java/edu/wpi/grip/core/cuda/CudaVerifierTest.java

```diff
@@ -0,0 +1,49 @@
+package edu.wpi.grip.core.cuda;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CudaVerifierTest {
```

Comment:
Cool!

Reply:
IOC and composition makes this a pretty trivial class 🙂